src/main/java/com/commafeed/backend/HttpGetter.java

```diff
@@ -223,4 +223,11 @@ public void checkServerTrusted(X509Certificate[] arg0, String arg1) throws Certi
 			return null;
 		}
 	}
+

```

Comment:
??? Why did you add a main in the HttpGetter class?

If it was to test some stuff, you should have removed it afterwards.